app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for i in range(len(people)):
```

Comment:
You don't really need this cycle. You can create instances of `Person` class inside list comprehension  in line 13.
```
result = [Person(people[i]["name"], people[i]["age"]) for ... ]
``` 
In this case, you will create a new person and add it to the list in one command
If you still have questions about it you can ask in the `py_jan22_questions_and_answers` chat